korean/girlfriend/girlfriend.md

```diff
@@ -0,0 +1,9 @@
+당신을 너무나도 당황하여, 놀란 마음을 진정시키기 위하여 여자친구를 생각해본다. 
+
+어떤 일이 일어나나요?
+
+[여자친구의 웃는 모습이 생각난다](smile.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.